app-ng/src/jobbo.lisp

```diff
@@ -0,0 +1,74 @@
+;;;; app-ng/src/jobbo.lisp
+;;;;
+;;;; author: appleby
+;;;;
+;;;; TODO(appleby): This file will be renamed or else merged with job.lisp once that PR lands. I've
```

Comment:
Did that PR land?

Reply:
Not yet:  #180. I probably should have linked to it in the comment :-)